### PR TITLE
Added Imgchecker and fixed images

### LINKED ISSUE #738: Travis: check image size
Create a travis script that checks for image sizes and fails if images are too big

### PR DESCRIPTION
**Pull request by ankitrgadiya**
Fixed  #738 
* Added Imgchecker.rb By Abishek V Ashok.
* Added script in .travis.yml
* Added gem 'fastimage' used in Imgchecker.rb
* Fixed Photos accordingly.

### CODE REVIEW

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Change the directory names and the images sizes this site has varieties...

**Comment by abishekvashok** [comment]
@ankitrgadiya fixed it @niccokunzmann @jig08 @hemantjadon we are ready to go, shall i rebase it or some one among you shall.

**Comment by ankitrgadiya** [comment]
@Abhi2424shek I did. I'm no longer afraid of rebase

**Comment by abishekvashok** [comment]
@ankitrgadiya sure I was'nt on PC to rebase.

**Comment by ankitrgadiya** [comment]
@Abhi2424shek ah okay no problem. And Thanks bro for helping me.

**Comment by ankitrgadiya** [comment]
Fixed #738 #671

**Comment by abishekvashok** [comment]
Sure bro  will always be there, when you need me, so
keep calm and expect me 😉

**Comment by abishekvashok** [comment]
Update the pr body see the bold ones 😉

File: scripts/ImgChecker/ImgChecker.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+gImgFiles = Dir["./img/students/**/*.*"]
+mentorsImgFiles = Dir["./img/mentors/**/*.*"]
+blogsImgFiles = Dir["./img/blogs/**/*.*"]
+privlyImgFiles = Dir["./img/privly/**/*.*"]
+
+# Defines the standard maximum dimensions for images in each section
+studentsImgSize = 240
+gImgSize = 240
+mentorsImgSize = 240
+blogsImgSize = 240
```
Comment (robbyoconnor) [inline scripts/ImgChecker/ImgChecker.rb:21]: Why 240x240 for blogs? Those aren't square...

Comment (robbyoconnor) [reply scripts/ImgChecker/ImgChecker.rb:21]: Assuming you mean blog screenshots...

Comment (abishekvashok) [reply scripts/ImgChecker/ImgChecker.rb:21]: Please check the current size in the repo sir, and the site it uses cards of 240x240 px

Comment (ankitrgadiya) [reply scripts/ImgChecker/ImgChecker.rb:21]: @robbyoconnor All images for Blog Posts are 240x240 and that's why [tests](https://travis-ci.org/fossasia/gci15.fossasia.org/builds/184755491) are passing. :-)

Comment (robbyoconnor) [reply scripts/ImgChecker/ImgChecker.rb:21]: Good work

**Review by robbyoconnor — APPROVED** [review]
LGTM.

**Comment by abishekvashok** [comment]
@robbyoconnor then merge it sir!

**Comment by robbyoconnor** [comment]
I'm gonna merge this -- but will probably refactor the Image Checker script.

File: scripts/ImgChecker/ImgChecker.rb (hunk 1 above)
Comment (ankitrgadiya) [reply scripts/ImgChecker/ImgChecker.rb:21]: Thanks

**Pull request merged**